main.go

```diff
@@ -1,10 +1,10 @@
 package main
 
 import (
-	"go_bootcamp_api/infrastructure/router"
-	//good practices convention internals/personal/external
+	"github.com/lscantillowl/academy-go-q42021/infrastructure/router"
 )
 
+// Main function to start the application and initialize the router
```

Comment:
Only exported function require a comment, exported functions, structs, interfaces are the ones you name with capital letter.

For example: main is not exported, but Main would be exported.

It is not always require to export objects, you may think of this as in Java, not exporting is like private (for the package itself) or with capital is like public, everyone could access this.